assets/styles.css

```diff
@@ -1 +1,7 @@
-form > * {display:block;}
\ No newline at end of file
+form > * {display:block;}
```

Comment:
Why did you add this along with Bootstrap?

Reply:
will create changes